### PR TITLE
DVCLive get step

### PR DESCRIPTION
**Pull request by daavoo**
P.R. for https://github.com/iterative/dvclive/pull/142

Just API Reference for now. It should be probably included somewhere in the user guide in the future

### CODE REVIEW

File: content/docs/dvclive/api-reference/get_step.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+while dvclive.get_step() < 3:
+    dvclive.log("metric", 0.9)
+    dvclive.next_step()
+```
+
+## Description
+
+DVCLive uses a internal property called `step` that it's associated to each
+metric logged with [`dvclive.log()`].
```
[the remark this answers is not in the document]
Comment (daavoo) [reply content/docs/dvclive/api-reference/get_step.md:22]: Copy that

[the remark this answers is not in the document]
Comment (daavoo) [reply content/docs/dvclive/api-reference/get_step.md:22]: Prioritized back to address in this sprint https://github.com/iterative/dvclive/issues/114

File: content/docs/dvclive/api-reference/get_step.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+import dvclive
+
+while dvclive.get_step() < 3:
+    dvclive.log("metric", 0.9)
+    dvclive.next_step()
+```
+
+## Description
+
+DVCLive uses an internal property called `step` that it's associated to each
```
Comment (dberenbaum) [inline content/docs/dvclive/api-reference/get_step.md:21]:
```suggestion
DVCLive uses an internal property called `step` that's associated to each
```

Small grammar suggestion.

However, maybe there's a better way to put this:

```
DVCLive uses the `step` to track the progress of each metric logged with [`dvclive.log()`].
```

File: content/docs/dvclive/api-reference/log.md [hunk 3]
```diff
@@ -46,8 +46,9 @@ environment variables (when used alongside `DVC`).
 
 - `val` - The value to be added in the `name` column of a new row.
 
-- `step` (`None` by default) - The value to be added in the `step` column of a
-  new row. If `None`, the value of the internal `_step` property will be used.
+- `step` (`None` by default) - The value to be added in the
+  [`step`](/doc/dvclive/api-reference/get_step) column of a new row. If `None`,
+  the value of the internal `step` property will be used.
```
Comment (dberenbaum) [inline content/docs/dvclive/api-reference/log.md:51]:
```suggestion
- `step` (`None` by default) - The value to be added in the `step` column of a
  new row. If `None`, the value of
  [`dvclive.get_step()`](/doc/dvclive/api-reference/get_step) will be used.
```

What do you think of this? I'd prefer to use the public methods than referencing internal properties if possible.

File: content/docs/dvclive/api-reference/get_step.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (shcheklein) [reply content/docs/dvclive/api-reference/get_step.md:22]: 🙏

**Review by shcheklein — APPROVED** [review]

File: content/docs/dvclive/api-reference/next_step.md [hunk 4]
```diff
@@ -25,7 +25,7 @@ selected in [`dvclive.init()`] and whether `DVC` is available or not.
 
 If `summary` is True, on each `dvclive.next_step()` call, DVCLive will generate
 a summary of the values previously logged with [`dvclive.log()`], and increase
-the `step` count.
+the [`step`](/doc/dvclive/api-reference/get_step) count.
```
Comment (dberenbaum) [inline content/docs/dvclive/api-reference/next_step.md:28]: There are some other places where `step` is mentioned. Should these all link to doc/dvclive/api-reference/get_step? This is not a rhetorical question, I'm honestly not sure what is best.

**Review by dberenbaum — APPROVED** [review]
Looks good! I have some minor questions/comments, but nothing that should block merging this.

File: content/docs/dvclive/api-reference/next_step.md (hunk 4 above)
Comment (daavoo) [reply content/docs/dvclive/api-reference/next_step.md:28]: Yes, I think those should all link to `get_step`.

**Pull request merged**